pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -0,0 +1,363 @@
+/*
```

Comment:
Can we move cpuset to under `pkg/kubelet/cm`? In general, the architecture I'm hoping to converge on is one where the Container manager is responsible for all resource management. In container manager, we have one for node and pod. Node container manager deals with handling overall CPU capacity, allocatable, availability (pressure), etc. 
PodContainerManager deals with CPU allocation, NUMA, device allocation, and any other performance optimizations per pod. There needs to be shared state between NodeContainerManager and per pod PodContainerManager objects.
Kubelet Node Lifecycle management will have to understand NodeContainerManager APIs only.
Kubelet Pod lifecycle management will have to interact with PodContainerManager APIs only.

Reply:
Moved and re-wired in the sub-PR.